source/sass/main.scss

```diff
@@ -254,13 +260,13 @@ button {
     }
 
     cursor: pointer;
-    background: #feeb34;
+    background: $nsf-yellow;
     font-weight: bold;
     border: none;
 
     &:hover {
       border: none;
-      background: #fee601;
```

Comment:
This is a slightly darker yellow, pulled from https://healthreport.staging.wpengine.com/v01/web-literacy/